src/main/webapp/app/quiz/overview/service/course-training-quiz.service.spec.ts

```diff
@@ -30,4 +36,19 @@ describe('CourseTrainingQuizService', () => {
         expect(req.request.method).toBe('GET');
         req.flush(mockQuestions);
     });
+
+    it('should submit submission for training', fakeAsync(() => {
+        const mockSubmission = new QuizSubmission();
+        const mockResult = new Result();
+        mockResult.id = 1;
+        mockResult.score = 10;
+        service.submitForTraining(mockSubmission, exerciseId).subscribe((res) => {
+            expect(res.body!.id).toBe(1);
+            expect(res.body!.score).toBe(10);
+        });
+
+        const req = httpMock.expectOne({ method: 'POST', url: `api/quiz/exercises/${exerciseId}/submissions/training` });
```

Comment:
You could consider staying consistent with the test before
```suggestion
        const req = httpMock.expectOne(`api/quiz/exercises/${exerciseId}/submissions/training`);
        expect(req.request.method).toBe('POST');
```